tests/test_table.py

```diff
@@ -2149,6 +2147,23 @@ def test_repr(self, test_tbl: catalog.Table, all_mpnet_embed: func.Function) ->
         )
         _ = v2._repr_html_()  # TODO: Is there a good way to test this output?
 
+        s = pxt.create_snapshot('test_snap', test_tbl)
```

Comment:
Also include an example of a snapshot over a view? (Or a mixed hierarchy of view -> snapshot -> view -> snapshot or some such?)

Also include an example of a non-pure snapshot since the code path for resolving ancestors is (slightly) different?

Reply:
Done.